mlrun/model_monitoring/model_monitoring_batch.py

```diff
@@ -911,6 +919,57 @@ def _update_drift_in_input_stream(
                 endpoint=endpoint_id,
             )
 
+    def _update_drift_in_prometheus(
+        self,
+        endpoint_id: str,
+        drift_status: DriftStatus,
+        drift_result: Dict[str, Dict[str, Any]],
+    ):
+        """Push drift metrics to Prometheus registry. Please note that the metrics are being pushed through HTTP
+        to the monitoring stream pod that writes them into a local registry. Afterwards, Prometheus wil scrape these
+        metrics that will be available in the Grafana charts.
+
+        :param endpoint_id:   The unique id of the model endpoint.
+        :param drift_status:  Drift status result. Possible values can be found under DriftStatus enum class.
+        :param drift_result:  A dictionary that includes the drift results for each feature.
+
+
+        """
+        stream_http_path = (
+            mlrun.mlconf.model_endpoint_monitoring.default_http_sink.format(
+                project=self.project
+            )
+        )
+
+        statistical_metrics = ["hellinger_mean", "tvd_mean", "kld_mean"]
+        metrics = []
+        for metric in statistical_metrics:
+            metrics.append(
+                {
+                    mlrun.common.model_monitoring.EventFieldType.ENDPOINT_ID: endpoint_id,
+                    mlrun.common.model_monitoring.EventFieldType.METRIC: metric,
+                    mlrun.common.model_monitoring.EventFieldType.VALUE: drift_result[
+                        metric
+                    ],
+                }
+            )
+
+        res = requests.post(
+            url=stream_http_path + "/monitoring-batch-metrics", data=json.dumps(metrics)
+        )
```

Comment:
we have a request-retryable http client on mlrun, wouldnt want to use it? see mlrun/utils/http.py